src/test/java/com/training/services/tests/GAAccountTest.java

```diff
@@ -7,55 +7,58 @@
 import org.junit.jupiter.api.Test;
 import java.util.HashMap;
 import java.util.Map;
-import io.restassured.RestAssured;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import static io.restassured.RestAssured.given;
-
 
 public class GAAccountTest extends BaseTest {
     Map<String, Object> headerMap = new HashMap();
     private Logger logger = LoggerFactory.getLogger(this.getClass());
 
+    /**
+     * testGuestAuthenticate():post requested with body to get the +ve response and parameters passed through Json
+     */
     @Test
     public void  testGuestAuthenticate(){
         String uid="testShrikant@api.com";
-        RestAssured.baseURI = map.get("URI");
-        Response authenticationResponse = given().
-                header(map.get("AppKeyHeader"), (map.get("AppKeyValue"))).
-                header(map.get("ContentTypeHeader"), (map.get("ContentTypeValue")))
-                .body("{\n" +
-                        " \"uid\": \""+uid+"\",\n" +
-                        " \"password\": \"Password1\"\n" +
-                        "}").post("/authentication/login")
-                .then()
-                .extract()
-                .response()
-                .as(Response.class);
+        Map<String, Object> headerMap = new HashMap();
+        headerMap.put(map.get("AppKeyHeader"), map.get("AppKeyValue"));
+        headerMap.put(map.get("ContentTypeHeader"), map.get("ContentTypeValue"));
```

Comment:
lets use this in before Each and, directly pass the variable headerMap to RestEngine.